src/backend/joanie/core/serializers.py

```diff
@@ -65,11 +68,14 @@ def get_position(self, target_course):
         Retrieve the position of the course related to its product_relation
         """
         product = self.context.get("product", None)
-        return (
-            target_course.product_relations.only("position")
-            .get(product=product)
-            .position
-        )
+        return target_course.product_relations.get(product=product).position
```

Comment:
why remove the "only" clause? I suppose it's because now you retrieve also `is_graded`. But in  this case should we check that the object is retrieved only once from db and if not use a different pattern to inflate these 2 fields?